Rollup 3.x.x

**Describe the bug**
This plugin is not compatible with Rollup 3.x.x

**To Reproduce**
Steps to reproduce the behavior:

1. Go to a project with rollup 3.2.2 as dependence (cd ~/my-projects/project-with-rollup3).
2. Install this plugin (npm i rollup-plugin-summary).
3. See error.

**Expected behavior**
Plugin installed without dependence errors.

**Screenshots**

```
$ npm i rollup-plugin-summary
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: roll3@1.0.0
npm ERR! Found: rollup@3.2.2
npm ERR! node_modules/rollup
npm ERR!   rollup@"^3.2.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer rollup@"^2.0.0" from rollup-plugin-summary@1.4.3
npm ERR! node_modules/rollup-plugin-summary
npm ERR!   rollup-plugin-summary@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/sergio/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/sergio/.npm/_logs/2022-10-17T16_10_40_935Z-debug-0.log
```

**Desktop (please complete the following information):**

- OS: Fedora Linux 36 (Workstation Edition)
- Node.js v16.14.0
- npm 8.19.1
- Rollup 3.2.2

--- a/README.md
+++ b/README.md
@@ -2,7 +2,6 @@
 [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 ![https://img.shields.io/github/languages/top/yousifalraheem/rollup-plugin-summary](https://img.shields.io/github/languages/top/yousifalraheem/rollup-plugin-summary)
-![https://img.shields.io/npm/l/rollup-plugin-summary](https://img.shields.io/npm/l/rollup-plugin-summary)
 ![npm bundle size](https://img.shields.io/bundlephobia/minzip/rollup-plugin-summary)
 [![codecov](https://codecov.io/gh/yousifalraheem/rollup-plugin-summary/branch/master/graph/badge.svg)](https://codecov.io/gh/yousifalraheem/rollup-plugin-summary)
 [![Release](https://github.com/yousifalraheem/rollup-plugin-summary/actions/workflows/release.yml/badge.svg)](https://github.com/yousifalraheem/rollup-plugin-summary/actions/workflows/release.yml)

